dsls/sql/src/main/java/org/apache/beam/dsls/sql/interpreter/BeamSQLFnExecutor.java

```diff
@@ -74,7 +79,7 @@ public BeamSQLFnExecutor(BeamRelNode relNode) {
    * {@link #buildExpression(RexNode)} visits the operands of {@link RexNode} recursively,
    * and represent each {@link SqlOperator} with a corresponding {@link BeamSqlExpression}.
    */
-  private BeamSqlExpression buildExpression(RexNode rexNode) {
+  static BeamSqlExpression buildExpression(RexNode rexNode) {
```

Comment:
I change it to `static` because this method is essentially static, it does not reference any instance variable. How do you think?